doc/design.md

```diff
@@ -218,3 +218,15 @@ Covers:
 * `feat~src-directory-override`
 
 Needs: impl, itest
+
+### No Source Location in Report if Csustom Source Path
+
+`dsn~no-src-location-in-report-for-custom-source-path~1`
+
+ECM adds the source position of each error message declaration to the error-code-report. That allows the catalog to generate links for jumping to the source location.
+
+For projects that specify a different source (typically generated code by delombok plugin) that's not possible, since the code ist just generated and not checked into the repository. For that reason we decided not to report the source location for projects that specify a sourcePath.
```

Comment:
```suggestion
For projects that specify a different source (typically code generated by the delombok plugin) that's not possible, since the code is just generated and not checked into the repository. For that reason we decided not to report the source location for projects that specify a `sourcePath`.
```